fluxv2.tf

```diff
@@ -0,0 +1,195 @@
+locals {
+
+  known_hosts = "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ=="
+  fluxv2 = merge(
+    {
+      enabled               = false
+      create_ns             = true
+      namespace             = "flux-system"
+      target_path           = "production"
+      network_policy        = true
+      version               = "v0.8.0"
+      github_url            = "https://github.com/marie/curie"
+      repository            = "curie"
+      repository_visibility = "public"
+      branch                = "main"
+      github_token          = var.fluxv2["github_token"]
```

Comment:
set to `null` or `""` depending on what works with validation, it will be set with the merge below if set in `var.fluxv2`